test/specs/scenarios/accessibility.js

```diff
@@ -164,12 +164,14 @@ export const accessibilityScenarios = async (lang = 'en_US') => {
         runAccessibilityTest(driver)
       })
 
-      it('should verify accessibility for the cross device submit screen', async () => {
+      // FIXME: consistently fails
```

Comment:
What is the failure?

Reply:
Still investigating why this is now failing. It's a bit odd since I don't think any of us recently made any changes for cross device.

Reply:
There seemed to be some inconsistency with the steps in the Cross Device scenario happy path test, so copied that over for this test. It's still failing with a timeout when it gets to `runAccessibilityTest(driver)`. Wonder if it's because of the tab switching.